scripts/fix-mjs.sh

```
@@ -0,0 +1,6 @@
+for file in ./lib/esm/*.js; do
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)